src/databases/mod.rs

```diff
@@ -18,3 +18,5 @@ mod sqlite;
 #[cfg(feature = "sqlite")]
 #[cfg_attr(docsrs, doc(cfg(feature = "sqlite")))]
 pub use sqlite::*;
+#[doc(hidden)]
```

Comment:
I would not hide the Documentation for this. =]

Reply:
Lol didn't realize I did that haha